tlcc_enclave/generate_mrenclave.sh

```diff
@@ -13,7 +13,7 @@ tmp_name=tmp_enc_hash
 go_name=mrenclave.go
 
 cd $build_dir
-sgx_sign gendata -enclave enclave.so -config $enclave_dir/Enclave.config.xml -out $tmp_name
+sgx_sign gendata -enclave enclave.so -config $enclave_dir/enclave.config.xml -out $tmp_name
```

Comment:
Good catch, but the bigger problem seems that we do no catch any errors and so it was happily running so far (just producing only an empty mrenclave).   Easiest fix for that seems to be o just add a 'set -e' in the beginning?

Also, given that the generate_mrenclave.sh is the same in ecc_enclave and tlcc_enclave, maybe we should move that to a common bin directory? 

Thinking of it, the method here also seems rather brittle, the way we do it in pdo is more robust:
- add to sgx_sign sign the '-dumpfile "${SIGNED_ENCLAVE_METADATA}"' option and then
- extract it with a more understandable like the ' VAR_MRENCLAVE=$$(perl -0777 -ne 'if (/metadata->enclave_css.body.enclave_hash.m:([a-fx0-9 \n]+)/) { $$eh = $$1; $$eh=~s/0x| |\n//g; $$eh=~tr/a-z/A-Z/; print "$${eh}\n"; }' $(CONTRACT_ENCLAVE_MRENCLAVE_META_FILE))' in target '$(CONTRACT_ENCLAVE_MRENCLAVE_C_FILE)' in pservice/Makefile

Reply:
This deserves a separate PR! I created an issue for that #77